content/en/post/2017-10-17-acme-support-in-apache-httpd.markdown

```diff
@@ -15,7 +15,7 @@ ACME support being built in to one of the world’s most popular Web servers, Ap
 
 The Apache httpd ACME module is called mod_md. It’s currently in the [development version of httpd](https://svn.apache.org/viewvc/httpd/httpd/trunk/modules/md/) and a plan is being formulated to backport it to an httpd 2.4.x stable release. The mod_md code is also [available on GitHub](https://github.com/icing/mod_md).
 
-<div style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/gNJUpzNNWMw?rel=0" style="border: none;" allowfullscreen></iframe></div>
```

Comment:
Why not keeping Youtube "nocookie" ?

Reply:
It still sets cookies if the user clicks to view the video. I figure it's a little neater and a little more privacy-protective to do this and be able to get rid of the entire domain in our CSP; but if you/others think the tradeoff isn't worth it, I'd be fine reverting that part of the commit.

Reply:
Personally I'm fine with that change, after further reflection I think it's better with it!